PopPUNK/__main__.py

```diff
@@ -470,16 +504,19 @@ def main():
             genomeNetwork = indivNetworks[min(rank_list)]
 
         # Ensure all in dists are in final network
-        networkMissing = set(map(str,set(range(len(refList))).difference(list(genomeNetwork.vertices()))))
+        vertex_list = set(get_vertex_list(genomeNetwork, use_gpu = args.gpu_graph))
```

Comment:
Actually I think a better change would be to make the network into a class and store the GPU property there. Then we can have multiple algorithms for CPU/GPU easily, don't have to keep passing the same data, and have functions to convert between graph types in the class